src/test/javascript/grouping.controller.test.js

```diff
@@ -918,6 +920,17 @@ describe("GroupingController", () => {
             expect(scope.isBatchImport).toBeTrue();
         });
 
+        it("should not call functions within $scope.addMembers if $scope.isAddingMembers is true", () => {
+            scope.isAddingMembers = true;
+            scope.addMembers("Include", uhIdentifiers);
+
+            spyOn(gs, 'getMemberAttributeResultsAsync');
+            spyOn(gs, 'getMemberAttributeResults');
```

Comment:
This completely went over my head but the spyOn's should be above `scope.addMembers()` because you want to start spying before you call the function. Otherwise the test will always pass if `scope.isAddMembers` was false.